src/testRunner/unittests/tsbuild/javascriptProjectEmit.ts

```diff
@@ -0,0 +1,353 @@
+namespace ts {
+    describe("unittests:: tsbuild:: javascriptProjectEmit:: loads js-based projects and emits them correctly", () => {
+        let projFs: vfs.FileSystem;
+        const { time, tick } = getTime();
+        before(() => {
+            projFs = loadProjectFromFiles({
+                "/src/common/nominal.js": utils.dedent`
+                    /**
+                     * @template T, Name
+                     * @typedef {T & {[Symbol.species]: Name}} Nominal
+                     */
+                    module.exports = {};
+                    `,
+                "/src/common/tsconfig.json": utils.dedent`
+                    {
+                        "extends": "../../tsconfig.base.json",
+                        "compilerOptions": {
+                            "composite": true
+                        },
+                        "include": ["nominal.js"]
+                    }`,
+                "/src/sub-project/index.js": utils.dedent`
+                    import { Nominal } from '../common/nominal';
+
+                    /**

```

Comment:
You would need to rebaseline these after you merge master as now it baselines errors and output so we don't have to hardcode those errors everytime those change.

Reply:
And sorry about the casing.. You want to change casing for `initial-Build` to `initial-build` as otherwise tests fail on linux